components/compliance-service/cmd/inspec_runner/inspec_runner.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/chef/automate/components/compliance-service/cmd/inspec_runner/platform"
+	"github.com/sirupsen/logrus"
```

Comment:
I know it is a pain, but I think we should try to avoid most external dependencies in this tool.  We would like this stub to have as little code as possible at runtime.